doc/nrf/ug_matter.rst

```diff
@@ -50,32 +50,122 @@ The |NCS|'s Multiprotocol Service Layer (MPSL) driver allows running Bluetooth L
    nRF Connect platform in Matter
 
 For detailed description, see the `nRF Connect platform overview`_ Matter documentation page.
+If you are new to Matter, check also the tutorials on `DevZone`_.
+
+Matter development environment
+******************************
+
+When you program any of the available Matter samples to the supported development kits, you automatically build Matter's nRF Connect platform.
```

Comment:
I'm not sure what "Matter's nRF Connect platform" refers to. If it means that the platform integration component in Matter then it's not enough to say that DK + Matter's nRF Connect platform form the Matter accessory. How about using "Matter  stack for nRF Connect platform"? E.g. "Building any of the available Matter samples involves building Matter stack for nRF Connect platform".

Reply:
Personally, I like your last proposal.

Reply:
Implemented.